content/games/factions/companies.md

```diff
@@ -132,6 +151,12 @@ Throwers (1)
 Unfeeling (0)
 : Automatically pass when testing Resolve but round up hits inflicted against this Group.
 
+| Taunt              | Check | Requirements                                                            | Effect                                                                                                                                                                                             |
+| :----------------- | :---: | :---------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `P3W P3W P3W`      |  6+   | Arclight Rifles/Blasters only. Must be able to Shoot an enemy Group.    | This Group becomes Reckless, except they must Shoot instead of Attack if they are able. They may reroll any dice when Shooting. If Attacked, they must reroll any dice that hit.                   |
```

Comment:
```suggestion
| `P3W P3W P3W`      |  6+   | Arclight Rifles/Blasters only. Must be able to Shoot an enemy Group.    | This Group becomes Reckless, except they must Shoot instead of Attack, if able. They may reroll any dice when Shooting. If Attacked, they must reroll any of their dice that hit.                   |
```

Reply:
quick note: i just want to make sure these apostrophes are the correct ones. they look funny to me, but i wouldnt be surprised if they were just displayed funny in github OR if they were some programmer joke.

Reply:
Yup! In markdown, wrapping text in backticks renders the text `like this`, which is monospaced code font. Because they're robots. 😣